assets/xml/objects/object_ge1.xml

```diff
@@ -1,63 +1,83 @@
+        <Animation Name="gGerudoWhiteTrudgingOffAnim" Offset="0x202C" /><!-- Original name: "gb_tobotobo" ("totteringly/trudgingly") -->
+        <Animation Name="gGerudoWhiteGreatBayCutsceneAnim" Offset="0x2148" /><!-- one frame used for each part of Great Bay Temple first entry cutscene. Original name: "gb_twister" -->
+        <Animation Name="gGerudoWhiteExcitedClappingAnim" Offset="0x2954" /><!-- maybe Original name: "gb_wakuwaku", ("excited") -->
+        <Animation Name="gGerudoWhiteUnfoldingArmsAnim" Offset="0x2AD8" /><!-- maybe "ge1_hanasi" ("talk") -->
+
+        <!-- From here to the end this file is identical to OoT's object_ge1, offset 0x0-0x9860 -->
+        <Animation Name="gGerudoWhiteArmsFoldedAnim" Offset="0x2B98" /><!-- Original name: "ge1_matsu" ("waiting") -->
+
+        <!-- White-clothed Gerudo Limbs -->
+        <Limb Name="gGerudoWhiteWaistLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_WAIST" Offset="0x2BB0" />
+        <Limb Name="gGerudoWhiteLeftThighLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_LEFT_THIGH" Offset="0x2BBC" />
+        <Limb Name="gGerudoWhiteLeftShinLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_LEFT_LOWER_LEG" Offset="0x2BC8" />
+        <Limb Name="gGerudoWhiteLeftFootLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_LEFT_FOOT" Offset="0x2BD4" />
+        <Limb Name="gGerudoWhiteRightThighLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_RIGHT_THIGH" Offset="0x2BE0" />
+        <Limb Name="gGerudoWhiteRightShinLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_RIGHT_LOWER_LEG" Offset="0x2BEC" />
+        <Limb Name="gGerudoWhiteRightFootLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_RIGHT_FOOT" Offset="0x2BF8" />
+        <Limb Name="gGerudoWhiteTorsoLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_TORSO" Offset="0x2C04" />
+        <Limb Name="gGerudoWhiteLeftUpperArmLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_LEFT_UPPER_ARM" Offset="0x2C10" />
+        <Limb Name="gGerudoWhiteLeftForearmLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_LEFT_FOREARM" Offset="0x2C1C" />
+        <Limb Name="gGerudoWhiteLeftHandLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_LEFT_HAND" Offset="0x2C28" />
+        <Limb Name="gGerudoWhiteRightUpperArmLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_RIGHT_UPPER_ARM" Offset="0x2C34" />
+        <Limb Name="gGerudoWhiteRightForearmLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_RIGHT_FOREARM" Offset="0x2C40" />
+        <Limb Name="gGerudoWhiteRightHandLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_RIGHT_HAND" Offset="0x2C4C" />
+        <Limb Name="gGerudoWhiteHeadLimb" Type="Standard" EnumName="GERUDO_WHITE_LIMB_HEAD" Offset="0x2C58" />
+
+        <!-- White-clothed Gerudo Skeleton -->
+        <Skeleton Name="object_ge1_Skel_002CA0" Type="Flex" LimbType="Standard" LimbNone="GERUDO_WHITE_LIMB_NONE" LimbMax="GERUDO_WHITE_LIMB_MAX" EnumName="GerudoWhiteLimb" Offset="0x2CA0" />
```

Comment:
Any reason to not name this `gGerudoWhiteSkel`?

Reply:
Oh, missed the obvious, whoops.